src/Symfony/Component/Finder/Iterator/RecursiveDirectoryIterator.php

```diff
@@ -75,7 +75,7 @@ public function current(): SplFileInfo
      * @throws AccessDeniedException
      */
     #[\ReturnTypeWillChange]
-    public function getChildren()
+    public function getChildren(): \RecursiveIterator
```

Comment:
Regarding this specific return type: We need to narrow it down further. See #42468 and #42469.

Reply:
ok, I'm then reverting here and we'll handle it separately